components/Footer.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from "styled-components/native";
+import { View, Text, Linking } from 'react-native';
+
+
+export const Footer = () => {
+    return (
+        <FooterContainer>
+            <FooterText>To make someone else's day, please click the link below and write about somehing that made you smile today</FooterText>
+            <FooterLink onPress={() => Linking.openURL('https://spring2022-happy-tweets.netlify.app/')}>Happy Tweets</FooterLink>
```

Comment:
I'm very impressed with this feature in your footer component. You show the user and fellow code reviewers on how to apply and work with linking URL. This is a component with a clean and easy understandable code to follow!